packages/utreexo/src/test.cairo

```diff
@@ -0,0 +1,43 @@
+use crate::stump::state::UtreexoStumpState;
+use crate::stump::proof::UtreexoBatchProof;
+use crate::stump::accumulator::StumpUtreexoAccumulator;
+use core::testing::get_available_gas;
+
+#[derive(Drop, Serde, Debug)]
+struct Args {
+    state: UtreexoStumpState,
+    proof: UtreexoBatchProof,
+    leaves_to_del: Array<felt252>,
+    leaves_to_add: Array<felt252>,
+    expected_state: UtreexoStumpState,
+}
+
+fn main(mut arguments: Span<felt252>, _flags: felt252) {
+    let mut gas_before = get_available_gas();
+
+    let Args { mut state, proof, leaves_to_del, leaves_to_add: _, expected_state } =
```

Comment:
It's not implemented yet, once it's ready it will be added after deletion.
Most of the tests are ignored now, next iteration is making sure they work